### ISSUE TITLE
network request failed

### ISSUE THREAD

**Issue by saremeskandary** [BlueWallet/BlueWallet#4117]
I had an old LN private key which it hadn't `@https://lndhub.io` and I got `network request failed`.

I it take me lot's of time to find the problem. please add a guide or checker that if `@https://lndhub.io` wasn't in the end of private key add it automatically or throw an error.

**Comment by a455bcd9** [comment]
Had the same issue: `import error`: `network request failed`.

Solution: instead of using the QR code, write down the string below it and add `@https://lndhub.io` at the end.

**Comment by elRafa** [comment]
@saremeskandary and @a455bcd9 -- thank you for posting, as I also had the same issue importing. This is still not fixed. @Overtorment, any way to make it check for a server and if none, offer a field for the user to select/add it?

**Comment by Kayevo** [comment]
Title: Can't Import Lightning Wallet Blue Wallet

I have a problem importing my wallet on the first time I needed to do that. After that problem, I even used Blue Wallet.
But it's clear to me the incompetence of the application to follow your OWN wallet importation patterns.
Thank you @saremeskandary and @a455bcd9 for offering this tip to insert "@https://lndhub.io" at the end of the URI.

So if you are reading this comment and still having problems importing your Lightning Network Wallet on Blue wallet, the URI/URL pattern to import wallets is: 

`lndhub://<your_key>@https://lndhub.io`

for example: 
`lndhub://12345:12345@https://lndhub.io`

So click add a wallet, select import wallet button, and insert this URI.

Note: iPhone application cannot import wallets when I'm posting this message. I imported it on my other phone using the Android system.
